GD32F4XX/standard_peripheral/Include/gd32f4xx_timer.h

```diff
@@ -624,7 +624,7 @@ void timer_deinit(uint32_t timer_periph);
 /* initialize TIMER init parameter struct */
 void timer_struct_para_init(timer_parameter_struct* initpara);
 /* initialize TIMER counter */
-void timer_init(uint32_t timer_periph, timer_parameter_struct* initpara);
+void timer_initialize(uint32_t timer_periph, timer_parameter_struct* initpara);
```

Comment:
There should be a common way to fix conflicting names, e.g. prefix with `__`

Reply:
Done